python/adelphi/bin/test-adelphi

```diff
@@ -74,13 +109,13 @@ def runtests(cassandra, python, pytest):
             # exiting all the things
             try:
                 tox.cmdline(tox_args)
-            except SystemExit:
-                pass
+            except SystemExit as exc:
+                exitCodes.append(exc.code)
```

Comment:
The exit code that would be returned by tox lives in SystemExit.code. We need to trap this to determine how many of our tox "invocations" resulted in successful tests.